src/app/system/manage-data-tables/create-data-table/create-data-table.component.scss

```diff
@@ -0,0 +1,7 @@
+.checkbox-padding {
```

Comment:
Make sure you add classes only if necessary and with meaning full names related to their function. Add a line at the end of the file, remove any class if unnecessary.

Reply:
These classes are necessary for proper alignment. Although you can suggest any other name if you think that will suit best for it.